### ISSUE TITLE
Package up MS MARCO doc data

### ISSUE THREAD

**Issue by lintool** [castorini/docTTTTTquery#27]
These data files for doc expansion:

```
gsutil cp gs://neuralresearcher_data/msmarco_doc/msmarco-docs.tsv.gz .
gsutil cp gs://neuralresearcher_data/doc2query_t5_msmarco_doc/data/1/predicted_queries_topk_sample00?.txt???-1004000 .
gsutil cp gs://neuralresearcher_data/doc2query_t5_msmarco_doc/data/1/segment_doc_ids.txt .
```
Can you please convert them into download links so I can mirror in Dropbox and GitLab?

Can we zip up `predicted_queries` into a single file to download?
BTW, is it easy to add "doc" somewhere in the name to disambiguate with the passage predictions?

**Comment by rodrigonogueira4** [comment]
Sorry about the late reply. Here you go:

```
https://storage.googleapis.com/neuralresearcher_data/msmarco_doc/msmarco-docs.tsv.gz
https://storage.googleapis.com/neuralresearcher_data/doc2query_t5_msmarco_doc/data/1/predicted_queries_doc.tar.gz
https://storage.googleapis.com/neuralresearcher_data/doc2query_t5_msmarco_doc/data/1/segment_doc_ids.txt 
```

I created a tar.gz because it is smaller than a zip.

**Comment by lintool** [comment]
Can you share MD5 checksums?

**Comment by rodrigonogueira4** [comment]
Sure:
```
103b19e21ad324d8a5f1ab562425c0b4  msmarco-docs.tsv.gz
4967214dfffbd33722837533c838143d  predicted_queries_doc.tar.gz
82c00bebab0d98c1dc07d78fac3d8b8d  segment_doc_ids.txt
```

**Comment by lintool** [comment]
Closed by #28
